github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -797,7 +798,7 @@ presubmits:
             memory: 4Gi
         securityContext:
           privileged: true
-  - always_run: true
+  - always_run: false
```

Comment:
This one is a candid to be phase 2
this why i changed this one
once plugin is tested, we can change all non latest k8s version / ipv6 etc to be phase 2
unless we want ipv6 to be considered phase 1, but imo not needed
whoever works on specific ipv6 feature, which is rare, will be able to /test manually during development
chosen it as it is one that doesnt change much, and we have the dual stack running as phase 1

Reply:
ok thanks @oshoval

Reply:
Is it fine that we require to add `# phase2` comment on those  ?
this way it will be more explicit they are phase2 , and will allow us to change them to true when
creating a new release

otherwise it means we need to read the config, and filter the jobs that need to be updated
as this plugin does real time, when we create a new release